libhb/param.c

```diff
@@ -115,6 +115,16 @@ static hb_filter_param_t chroma_smooth_tunes[] =
     { 0, NULL,          NULL,         NULL              }
 };
 
+static hb_filter_param_t colorspace_presets[] =
+{
+    { 1, "Custom",          "custom",       NULL        },
+    { 2, "BT.2020",         "bt2020",       "primaries=bt2020:matrix=bt2020ncl:transfer=bt2020-10:tonemap=hable:desat=0" },
+    { 2, "BT.709",          "bt709",        "primaries=bt709:matrix=bt709:transfer=bt709:tonemap=hable:desat=0"          },
```

Comment:
"BT.2020" and "BT.709" are both 2. Should't this be a different number?

Reply:
Yes, fixed.